config/jobs/kubernetes-sigs/special-resource-operator/special-resource-operator-presubmits.yaml

```diff
@@ -57,18 +21,6 @@ presubmits:
       description: "unit-tests the source code of special-resource-operator."
     spec:
       containers:
-      - image: golang:1.17
-        command:
-        - scripts/test-infra/unit_tests.sh
-  - name: pull-special-resource-operator-e2e-k8s
-    always_run: true
-    decorate: true
-    annotations:
-      testgrid-dashboards: sig-node-special-resource-operator
-      testgrid-tab-name: e2e-k8s
-      description: "e2e-k8s the source code of special-resource-operator."
-    spec:
-      containers:
-      - image: golang:1.17
+      - image: golang:1.18
         command:
-        - scripts/test-infra/e2e_k8s.sh
+        - ci/prow/unit_tests.sh
```

Comment:
I would remove the extension here as well.

Reply:
Fixed.